tests/execute/results-custom/test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    testName="/test/custom-results"
+    rlPhaseStartTest "${testName}"
+        rlRun "run=\$(mktemp -d)" 0 "Creating run direcotory/id for ${testName}"
```

Comment:
Instead of creating/removing the `run` directory five times, what about creating it just once in the setup phase and using `tmt run --scratch` to cleanup previously created workdir?